components/contribution-flow/ContributionFlowStepContainer.js

```diff
@@ -11,6 +11,7 @@ import StyledHr from '../StyledHr';
 import { H4 } from '../Text';
 import { withUser } from '../UserProvider';
 
+import CryptoStepDetails from './CryptoStepDetails';
```

Comment:
`StepDetailsCrypto` would be better I think.

Reply:
Done. 👍🏽